torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -1246,5 +1244,54 @@ c10::optional<const Node*> AliasDb::getLastWildcard() const {
     return c10::nullopt;
   }
 }
+
+TORCH_API bool aliasAnalysisHasSpecialCaseFor(Symbol symbol) {
+  // WARNING: by adding a case to this list, you are asserting that you have
+  // added a case for the unschematized node in AliasDb::analyze
+  const static std::unordered_set<Symbol> handled = {
```

Comment:
Could you add an assertion at 
https://github.com/pytorch/pytorch/blob/d70b7d18ff781cc54d7bd10dc306503f53e7a015/torch/csrc/jit/passes/alias_analysis.cpp#L664

that the op is not in handled so that the two sets are forced to be equal ?